.github/workflows/Suppress-Trivy.yml

```diff
@@ -0,0 +1,59 @@
+name: Suppress-Trivy - write to file
+
+on:
+  issue_comment:
+    types:
+      - created
+  workflow_dispatch:
+    inputs:
+      cve_id:
+        description: CVE ID
+        type: string
+        required: true
+      reason:
+        description: Reason
+        type: string
+        required: true
+      project:
+        description: Which project
+        required: true
+
+jobs:
+  build:
+    if: github.event_name=='workflow_dispatch' || (startsWith(github.event.comment.body, 'Suppress') && startsWith(github.event.issue.title, 'CVE-'))
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+        with:
+          ref: main
+
+      - name: Set CVE from Issue Title automatically
+        if: github.event_name=='issue_comment'
+        run: |
+          echo cve_id=${{ github.event.issue.title }} >> $GITHUB_ENV
+          echo reason=${{ github.event.comment.body }} >> $GITHUB_ENV
+          echo project=${{ github.event.label.name }} >> $GITHUB_ENV
```

Comment:
Is there a reason for using a label here, instead of including the project name in the title?  I'm just wary that we'll have to create new labels for each of our projects as we add them.